src/react-native.js

```diff
@@ -82,7 +82,7 @@ const ReactNative = {
   StyleSheet: require('./api/StyleSheet'),
   TimePickerAndroid: require('./api/TimePickerAndroid'),
   UIManager: require('./NativeModules/UIManager'),
-  VibrationIOS: require('./api/VibrationIOS'),
+  Vibration: require('./api/Vibration'),
```

Comment:
this should arguably be a part of a different PR, and maybe go along with some sort of version scheming or something. I'm not sure what the right thing for us to be doing here is long term...  It may serve us well to have version schemes that mimic RN core's?  Perhaps topic for another day.